docs/Module3/5_ELN_using_built-in_schema/M3_2_1_creating_entries_built-in_schema.md

```diff
@@ -1,29 +1,29 @@
-# Creating NOMAD entries using the ELN built-in ELN schemas
-In this section, you will learn how to use NOMAD's built-in ELN schemas to create entries of entities and activities in your experiment.
+# Creating NOMAD Entries Using the ELN Built-in ELN Schemas
+In this section, you will learn how to use NOMAD's built-in ELN schemas to create entries for entities and activities in your experiment.
 
-After planning for your experiments, you would start by documenting your experimental work. This involves creating a record of all the materials and instruments that will be used during your experiment, or **entities**. In addition, the conditions under which the materials will be processed and the methods of measurement will be outlined, i.e., **activities**
+After planning for your experiments, you would start by documenting your experimental work. This involves creating a record of all the materials and instruments that will be used during your experiment, i.e., **entities**. In addition, you should outline the conditions under which the materials will be processed and the methods of measurement, i.e., **activities**.
 
-This record in NOMAD is made by creating entries for each entity and activity that will be used in your experiments.
+In NOMAD, this is done by creating entries for each entity and activity that will be used in your experiments.
 
 ![Alt text](../images/overview/3.png)
 
 
 
-### **Step 1:** Create a new upload for the electronic lab note book for the experiments. This upload will be your ELN of the planned experiment. 
+### **Step 1:** Create a New Upload for the Electronic Lab Notebook for the Experiments. This Upload will be Your ELN of the Planned Experiment. 
 
-![1- Create a new upload for the electronic lab note book for the experiments.](../images/ELN_builtin_schema/1.png)
+![1- Create a new upload for the electronic lab notebook for the experiments.](../images/ELN_builtin_schema/1.png)
 
-![2- Create a new upload for the electronic lab note book for the experiments.](../images/ELN_builtin_schema/2.png)
+![2- Create a new upload for the electronic lab notebook for the experiments.](../images/ELN_builtin_schema/2.png)
     
-![3- Create a new upload for the electronic lab note book for the experiments.](../images/ELN_builtin_schema/3.png)
+![3- Create a new upload for the electronic lab notebook for the experiments.](../images/ELN_builtin_schema/3.png)
     
 ![4- Click on the pen icon to provide a name for your upload](../images/ELN_builtin_schema/4.png)
     
 ![5- Provide a descriptive name for your upload](../images/ELN_builtin_schema/5.png)
     
 ![6- Click save](../images/ELN_builtin_schema/6.png)
 
-### **Step 2:** Create an **entry** using NOMAD's built-in schema.
+### **Step 2:** Create an **entry** Using NOMAD's Built-in Schema.
```

Comment:
Lets remove **Step 2** , and use the steps number in the images to refer to individual steps. What do you think?